### PR TITLE
FYLE-cqkder: Fixed the spacing of loader and text in delete pop up

### PR DESCRIPTION
**Pull request by MasterYushi**
### Before: 
image

### After: 
image

### CODE REVIEW

File: src/global.scss [hunk 1]
```diff
@@ -357,7 +357,7 @@ textarea:focus {
   height: 14px;
   display: inline-block;
   vertical-align: middle;
-  margin-left: 4px;
+  margin-inline: 4px;
```
Comment (abhishek1234321) [inline src/global.scss:360]: the browser support for this isn't great https://developer.mozilla.org/en-US/docs/Web/CSS/margin-inline, could you check if what browser does ionic 5 / capacitor uses and check if this property is supported there

if not, we should stick with margin-left or right,

**Review by abhishek1234321 — CHANGES_REQUESTED** [review]

File: src/global.scss (hunk 1 above)
Comment (MasterYushi) [reply src/global.scss:360]: ### I'm not sure. This is the only info I could find in the docs: 
image

### But I'll change it to `margin-left` and `margin-right` to be safe.

File: src/global.scss [hunk 2]
```diff
@@ -358,6 +358,7 @@ textarea:focus {
   display: inline-block;
   vertical-align: middle;
   margin-left: 4px;
+  margin-right: 4px;
```
Comment (Dimple16) [inline src/global.scss:361]: `margin: 0 4px;`

Comment (Dimple16) [reply src/global.scss:361]: can remove these:
```
margin-left: 4px;
margin-right: 4px;
```

Comment (Dimple16) [reply src/global.scss:361]: refer https://developer.mozilla.org/en-US/docs/Web/CSS/margin for margin shorthand properties @MasterYushi

**Review by Dimple16 — APPROVED** [review]

**Review by abhishek1234321 — APPROVED** [review]

**Pull request merged**